Library/Homebrew/cmd/info.rb

```diff
@@ -45,6 +45,8 @@ class Info < AbstractCommand
         switch "--github",
                description: "Open the GitHub source page for <formula> and <cask> in a browser. " \
                             "To view the history locally: `brew log -p` <formula> or <cask>"
+        switch "--github-manifest",
```

Comment:
Does this work with --json?

Reply:
Not yet. Haven't had time to look at that code path. Also need to figure out the path to get information to formulae.brew.sh, which may be similar.

Reply:
Yeh, it's that code path. If it doesn't work for now: might be worth marking as conflicting?

Reply:
I will try to look into JSON API path before merge.

For accurate info, could fill in data as part of bottle files when outputting `--variations` (would require fetching every manifest, but this is usually not run by users), e.g.
```
            "arm64_ventura": {
              "cellar": "/opt/homebrew/Cellar",
              "url": "https://ghcr.io/v2/homebrew/core/mesa/blobs/sha256:4d6107ebd85c3e25882508c93c1b827906bb186ef4c1107af849f52a6f7f9c1c",
              "sha256": "4d6107ebd85c3e25882508c93c1b827906bb186ef4c1107af849f52a6f7f9c1c",
              "bottle_size": 6434657,
              "installed_size": 28148482
            },
            "sonoma": {
              "cellar": "/usr/local/Cellar",
              "url": "https://ghcr.io/v2/homebrew/core/mesa/blobs/sha256:1c2e1cee7a201517a132a73ce1c63c84ec1eee281688f99f95a581a7290d978e",
              "sha256": "1c2e1cee7a201517a132a73ce1c63c84ec1eee281688f99f95a581a7290d978e",
              "bottle_size": 6706819,
              "installed_size": 27830789
            }
```

Still haven't figured out a way to load this information back in to avoid fetching manifest again.

---

Alternative would be an approximation at top-level by picking one of the bottles to base size off of. One disadvantage here is a large under-approximation on Linux for things like Perl formulae (given macOS perl has pre-installed multiple packages).

Reply:
Actually, may just add conflict for now and create a follow up PR to work on JSON API.

Reply:
> For accurate info, could fill in data as part of bottle files when outputting `--variations` (would require fetching every manifest, but this is usually not run by users)

I think this would make sense to do 👍🏻